src/main/java/longtimenosee/logic/commands/PolicyAddCommand.java

```diff
@@ -16,11 +16,11 @@
 public class PolicyAddCommand extends Command {
     public static final String COMMAND_WORD = "addPolicy";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a policy to the LTNS. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a policy to the LTNS. \n"
             + "Parameters: "
             + PREFIX_TITLE + "TITLE "
             //Todo: Replace message with a command that lists out different companies
-            + PREFIX_COMPANY + "'One of: {MNF, PRU, AXA, GEL, NTU, ETQ, TML, AIA, AVI, FWD}'"
```

Comment:
![image](https://user-images.githubusercontent.com/97422591/199453356-0d456c89-70e2-48ba-bd5a-f3d45fddf521.png)

Spacing seems odd. I think there's a spacing missed out between cmp and cms